website/docs/topics/large-segments.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Dealing with large constraint lists
+---
+
+import Figure from '@site/src/components/Figure/Figure.tsx'
+
+:::info
+
+In this explanatory guide, we will discuss how best to deal with large and complex constraints, such as when you want to constrain a feature to a list of 150 user IDs.
+
+:::
+
+Unleash offers several ways to limit feature exposure to a specified audience, such as the [User IDs strategy](../reference/activation-strategies.md#userids), [strategy constraints](../reference/strategy-constraints.md), and [segments](../reference/segments.mdx). Each of these options make it easy to add some sort of user identifier to the list of users who should get access to a feature.
+
+Because of their availability and ease of use with smaller lists, it can be tempting to just keep adding identifiers to those lists. However, once you start approaching a hundred elements we recommend that you find another way to manage these IDs. In fact, it's probably better to stop well before you get that far.
+
+The rest of this document will explain why this is generally a bad idea and how we suggest you solve your problems instead. However, as always there **are** exceptional cases where this is the solution.
+
+How large is too large? How complex is too complex? It depends. However, smaller and simpler is generally better.
+
+## The cost of data
+
+First, let's talk a bit about Unleash's architecture: **Your Unleash instance**is where you define all your features, their strategies, and constraints (and a lot more). However, the Unleash instance itself does not do any of the feature evaluation[^1]. That responsibility is relegated to the Unleash SDKs (or [Edge](/generated/unleash-edge.md) / the [Unleash proxy](/generated/unleash-proxy.md)).
```

Comment:
Delegated maybe? It's an important responsibility. Entirely optional suggestion

```suggestion
First, let's talk a bit about Unleash's architecture: **Your Unleash instance**is where you define all your features, their strategies, and constraints (and a lot more). However, the Unleash instance itself does not do any of the feature evaluation[^1]. That responsibility is delegated to the Unleash SDKs (or [Edge](/generated/unleash-edge.md) / the [Unleash proxy](/generated/unleash-proxy.md)).
```

Reply:
+1 for delegated here

Reply:
Yeah, I'm good with delegated. Good suggestion 🙌🏼